website/server/models/newsPost.js

```diff
@@ -0,0 +1,84 @@
+import mongoose from 'mongoose';
+import baseModel from '../libs/baseModel';
+import logger from '../libs/logger';
+
+const { Schema } = mongoose;
+const POSTS_PER_PAGE = 10;
+
+export const schema = new Schema({
+  title: { $type: String },
```

Comment:
this isn't super important given access to the routes will be limited but there's no check nor here nor at the routes level for required fields or things that cannot be updated. Not sure if it's something we want to check for